src/index.html

```diff
@@ -160,35 +158,190 @@
             <ul class="p-newjobs__list__li__salary">
               <li class="p-newjobs__list__li__salary__parts">
                 <p class="p-newjobs__list__li__salary__parts__title">給与</p>
-                <img class="p-newjobs__list__li__salary__parts__text" src="./img/picture/newjobs-text01.png" alt="月給 148,000円～">
+                <p class="p-newjobs__list__li__salary__parts__text">月給 148,000円～</p>
               </li>
               <li class="p-newjobs__list__li__salary__parts">
                 <p class="p-newjobs__list__li__salary__parts__title">所在地</p>
-                <img class="p-newjobs__list__li__salary__parts__text" src="./img/picture/newjobs-text02.png" alt="三重県伊勢市">
+                <p class="p-newjobs__list__li__salary__parts__text">三重県伊勢市</p>
               </li>
               <li class="p-newjobs__list__li__salary__parts">
                 <p class="p-newjobs__list__li__salary__parts__title">最寄駅</p>
-                <img class="p-newjobs__list__li__salary__parts__text" src="./img/picture/newjobs-text03.png" alt="伊勢市駅徒歩15分">
+                <p class="p-newjobs__list__li__salary__parts__text">伊勢市駅徒歩15分</p>
               </li>
             </ul>
           </li>
         </ul>
         <button class="p-newjobs__carousel-button_left c-carousel-button"></button>
         <button class="p-newjobs__carousel-button_right c-carousel-button"></button>
       </div>
-      <a class="p-newjobs__button__link" href="" ><div class="p-newjobs__button c-button">
-        新着求人一覧をもっと見る
-      </div></a>
+      <a class="p-newjobs__button__link" href="">
+        <div class="p-newjobs__button c-button">
+          新着求人一覧をもっと見る
+        </div>
+      </a>
     </div>
 
     <div class="p-contents">
       <div class="p-contents__img-dark">
-      <div class="p-contents__font-area">
-        <p class="p-contents__font">ギズジョブなら理想の職場がきっと見つかる</p>
-        <p class="p-contents__font">ギズジョブは完全無料の求職者向け求人サービスです。</p>
+        <div class="p-contents__font-area">
+          <p class="p-contents__font">ギズジョブなら理想の職場がきっと見つかる</p>
+          <p class="p-contents__font">ギズジョブは完全無料の求職者向け求人サービスです。</p>
+        </div>
       </div>
     </div>
+
+    <div class="p-features">
+
+        <div class="p-features__contents">
+          <h2 class="p-features__contents__title">ギズジョブの4つの特徴</h2>
+          <ul class="p-features__contents__detailflex">
+            <li class="p-features__contents__detail">
```

Comment:
以下がシンプルで良いかと思いましたがいかがでしょう？

- 「flex」などのスタイルに依存した命名をしてしまうとスタイルの変更があって flex じゃなくなった時にセレクタ名の変更も必要になってしまう（今回の場合例が極端かもしれませんが）
- list (ul) のなかに1つ1つの item (li) があるので、複数形・単数形で表現しちゃった方が読みやすいかなと思いました

```suggestion
          <ul class="p-features__contents">
            <li class="p-features__content">
```

Reply:
一つ上の指摘でp-jobsearch以外の各範囲の2つ目のdivタグに付けている".〇〇__contents"を".l-inner"に修正しましたので、現状ul,liタグで"〇〇__detailflex"、"〇〇__detail"になっているものを"〇〇__contents"、"〇〇__content"になるよう修正します。
スタイルの変更があった際にクラス名を修正することになることを視野に入れていなかったため、今後クラス名の定義は気を付けます！